### PR TITLE
Init

### PR DESCRIPTION
**Pull request by pampatzoglou**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                 type: 'ONE_TO_ONE_NAT'
+       zone: "{{ zone }}"
+       project: "{{ gcp_project }}"
+       auth_kind: "{{ gcp_cred_kind }}"
+       service_account_file: "{{ gcp_cred_file }}"
+       scopes:
+         - https://www.googleapis.com/auth/compute
+   register: instance
+
+ - name: create working directory for benchmarking tool
```
Comment (fgimenez) [inline tasks/main.yml]: We should not run the benchmarks on this role, the idea was to reuse the role from Fabio and create an additional one.

File: w3f/substrate-benchmarks/plugins/packet_net.ini [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Ansible Packet.net external inventory script settings
```
Comment (fgimenez) [inline w3f/substrate-benchmarks/plugins/packet_net.ini]: Why do we have packet config here? We need to remove it.

File: w3f/substrate-benchmarks/plugins/packet_net.py [hunk 3]
```diff
@@ -0,0 +1,506 @@
+#!/usr/bin/env python
```
Comment (fgimenez) [inline w3f/substrate-benchmarks/plugins/packet_net.py]: Please, remove all the files not related to GCP.

File: w3f/substrate-benchmarks/roles/aws/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (fgimenez) [inline w3f/substrate-benchmarks/roles/aws/.travis.yml]: Please, remove all the files not related to GCP.

**Review by fgimenez — CHANGES_REQUESTED** [review]
Please remove all the code not related to GCP, let's focus on the goal. Also we need to remove Fabio's role and add a simple script to clone it from the remote repo.

File: w3f/substrate-benchmarks/plugins/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Collections Plugins Directory
```
Comment (fgimenez) [inline w3f/substrate-benchmarks/plugins/README.md]: We could completely remove the plugins directory, only packet files are here.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+pip install -r requirements.txt
+$ ssh-keygen -o -a 100 -t ed25519 -f ~/.ssh/id_ed25519_gcp_compute_gcp-ansible -C "first.last@domain.com"
```
Comment (fgimenez) [inline README.md]: Why not using this ansible module for creating the keys? https://docs.ansible.com/ansible/latest/modules/openssh_keypair_module.html

Comment (pampatzoglou) [reply README.md]: we are
https://github.com/w3f/substrate-benchmarks/blob/init/w3f/substrate-benchmarks/roles/gcp/tasks/create.yml#L68-L73

**Review by fgimenez — APPROVED** [review]
Looks good! We are missing the CI config, but that can be handled on a separate PR.

**Pull request merged**